iotiletest/iotile/mock/devices/report_test_device.py

```diff
@@ -43,9 +43,11 @@ class ReportTestDevice(VirtualIOTileDevice):
                 stream_id (int): The stream that this reading should be sent out
     """
 
-    def __init__(self, args):
+    def __init__(self, args, module_name=None):
```

Comment:
I would put `module_name` as a parameter in args (with the default of 'Rptdev' if not passed) since I don't think there's any way to pass arguments to this constructor except via the args dictionary.